### ISSUE TITLE
bcstats issue (merge error)

### ISSUE THREAD

**Issue by ckrishanu** [vikjam/bcstatsR#5]
Please see the following error

```
# Load bcstatsR
> library(bcstatsR)
> 
> # Load a toy example bundled with bcstatsR
> data(survey, bc)
> 
> # Compare the differences with bcstats
> compute.bc <- bcstats(surveydata = survey,
+                       bcdata     = bc,
+                       id         = "id",
+                       t1vars     = "gender",
+                       t2vars     = "gameresult",
+                       t3vars     = "itemssold",
+                       enumerator = "enum",
+                       ttest      = "itemssold")
Error in fix.by(by.y, y) : 'by' must specify a uniquely valid column
> 
> print(compute.bc$backcheck)
Error in print(compute.bc$backcheck) : object 'compute.bc' not found
> 
```

**Comment by vikjam** [comment]
Hi @ckrishanu, thanks for taking a look! I saw that bug this morning and fixed it my [last commit](https://github.com/vikjam/bcstatsR/commit/3c9b980963dfb77e8b5dcd3fa8ee76f1a7680905). Try reinstalling to get the update.

```
detach('package:bcstatsR', unload = TRUE)
devtools::install_github('vikjam/bcstatsR')
```

If you're still experiencing the error. Please let me know!

**Comment by ckrishanu** [comment]
Hi @vikjam , the error is solved. I checked. Thanks. :-)

**Comment by vikjam** [comment]
Resolved in commit 3c9b980963dfb77e8b5dcd3fa8ee76f1a7680905.
